components/ProfileCard.js

```diff
@@ -28,10 +34,16 @@ const ProfileCard = () => {
     <View style={styles.cardContainer}>
       <View style={styles.profileImageContainer}>
         <Image
-          source={require('../assets/images/user.png')} // Replace with your image URL
+          source={require('../assets/images/user.png')}
           style={styles.profileImage}
         />
       </View>
+      <TouchableOpacity onPress={handleUserSettingsPress} style={styles.modalButton}>
+        <Image
+          source={require('../assets/images/settings-mobile.png')}
```

Comment:
I believe we can require these at the top and name them like a variable to be used here so it is easier to replace them if we move them

Reply:
Any paths stress me out because things can move and then break without us knowing. I do not know the best way to handle them though

Reply:
I'll look into this and work on cleaning this up today.